src/inventory/gilded_rose.js

```diff
@@ -21,59 +21,99 @@ const items = [
 
 updateQuality(items);
 */
+function checkNotSpecialItem(item) {
+  return (item.name != 'Aged Brie' &&
+    item.name != 'Backstage passes to a TAFKAL80ETC concert' &&
+    item.name != 'Sulfuras, Hand of Ragnaros')
+};
+function decreaseQuality(item, amount = 1) {
+  item.quality -= amount;
+}
+function increaseQuality(item, amount = 1) {
+  item.quality += amount;
+}
+function decreaseSellIn(item, amount = 1) {
+  item.sell_in -= 1;
+}
+function standardItemChanges(item) {
+  // then check that the quality between 0-50, and decrease the quality by 1
+  if (item.quality > 0 && item.quality < 50) {
+    decreaseQuality(item, 1);
+    // if the sell_in becomes negative, decrease quality again by 1
+    if(item.sell_in < 0) {
+      decreaseQuality(item, 1);
+    }
+  }
+}
+function agedBrieQuality(item) {
+  const { quality } = item;
+  switch (true) {
+    // if the quality is between 0-50 && the sell_in is negative, increase the brie quality by 2
+    case quality > 0 && quality < 50 && item.sell_in < 0:
+      increaseQuality(item, 2);
+      break;
+    // if the quality is between 0-50, increase the brie quality by 1
+    case quality > 0 && quality < 50: 
```

Comment:
and this would just be your default case
```suggestion
    default: 
```

Reply:
All of these comments that are all related, about the switch predicate, make sense in my head. However, when I made your suggested changes and reran my tests, I got a failing test (about the item's quality not being over 50). I figured out how to pass that test by adding extra `if`s in my cases, but that felt too deep. 

I ended up getting that failing test to pass by wrapping the whole switch table in an if statement (using `quality > 0 && quality < 50`).